File: cpp/Ram.cpp
#include <iostream>

using namespace std;

class Ram
{
private:
    int capacity;
    int price;

public:
    Ram() {}
    Ram(int capacity, int price)
    {
        this->capacity = capacity;
        this->price = price;
    }

    int getCapacity()
    {
        return this->capacity;
    }
    void setCapacity(int capacity)
    {
        this->capacity = capacity;
    }

    int getPrice()
    {
        return this->price;
    }
    void setPrice(int price)
    {
        this->price = price;
    }

    void printRam()
    {
        cout << "Ram Capacity: " << this->capacity << " GB\n";
        cout << "Ram Price: " << this->price << endl;
    }
    ~Ram() {}
};

File: python/Ram.py
class Ram:
    def __init__(self, capacity = 0, price = 0):
        self.capacity = capacity
        self.price = price
    
    def getCapacity(self):
        return self.capacity

    def setCapacity(self, capacity):
        self.capacity = capacity

    def getPrice(self):
        return self.price
    
    def setPrice(self, price):
        self.price = price
    
    def printRam(self):
        print("Ram Capacity: {} GB".format(self.capacity))
        print("Ram Price: {}".format(self.price))
